static/js/utils/tile_utils.tsx

```diff
@@ -27,6 +27,7 @@ export interface ReplacementStrings {
   statVar?: string;
   xDate?: string;
   yDate?: string;
+  place_dcid?: string;
```

Comment:
nit: placeDcid

Maybe also update place to placeName or add a comment to say it's the name

Reply:
done. will add a todo to rename place and update it in a followup